test/known_issues/test-inspector-instrumentation-breakpoint.js

```diff
@@ -0,0 +1,54 @@
+// This test validates inspector's Debugger.setInstrumentationBreakpoint method.
+// Refs: https://github.com/nodejs/node/issues/31138
+
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
```

Comment:
It looks like skipping makes the test succeeds, which is considered a failure for `known_issues/` tests..